lib/pages/chat/chat.dart

```diff
@@ -1041,10 +1049,13 @@ class ChatController extends State<Chat>
     }
   }
 
-  void clearSelectedEvents() => setState(() {
-        selectedEvents.clear();
-        showEmojiPicker = false;
-      });
+  void clearSelectedEvents() {
+    emojiPickerNotifier.value = false;
+
+    setState(() {
```

Comment:
please don't setState

Reply:
setState for old logic, I know but can't update everything because the logic is too big.

<img width="579" alt="Screenshot 2023-09-13 at 17 42 32" src="https://github.com/linagora/twake-on-matrix/assets/99852347/084370c8-f19d-4bd2-b0b6-b62b6272582c">